src/js/media-store/media-store.ts

```diff
@@ -495,7 +492,8 @@ const createMediaStore = ({
       // For any state change request "actions"/"events" of media (and related) state,
       // these are handled by the `RequestMap`, which defines a function for a given change request type
       // that is responsible for what should happen as a result
-      if (requestMap[type]) {
+      // If a fatal error occurred, we should not process any more state change requests.
```

Comment:
suggestion(non-blocking):

```suggestion
      // If a fatal error occurred, we should not process any more state change requests,
      // but we should process updates to state owners or options/defaults, handled below,
      // which is why we don't simply early bail here.
```